visio_gui.py

```
@@ -0,0 +1,537 @@
+            # For direct approach, we assume success if no errors
+            if direct_result.returncode == 0:
+                deleted_count = len(file_paths)
+                failed_count = 0
+            else:
+                # If there are errors, assume all failed
+                deleted_count = 0
+                failed_count = len(file_paths)
+                
+            self.root.after(0, self._delete_complete, deleted_count, failed_count)
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)